src/components/DefineCompany/YourCompany.vue

```diff
@@ -14,8 +14,8 @@
               <v-chip
                 v-if="companyNameChanges"
                 class="ma-2 pa-2"
-                color="primary"
-                pill
+                color="#1669BB"
```

Comment:
If you set a named color (not "primary") then you have to set the text-color property as well ("white").

Reply:
ah. i see! Fixed in next commit